WINDOWS.md

```diff
@@ -0,0 +1,289 @@
+
+# Windows
+
+Instructions to run the wallet on Windows.
+
+## Install Android Studio
+
+See the [releases](https://developer.android.com/studio) that most fits for your use:
+
+- Windows(64-bit) [android-studio-2021.3.1.17-windows.exe](https://redirector.gvt1.com/edgedl/android/studio/install/2021.3.1.17/android-studio-2021.3.1.17-windows.exe)
+- Windows(64-bit) [android-studio-2021.3.1.17-windows.zip](https://redirector.gvt1.com/edgedl/android/studio/ide-zips/2021.3.1.17/android-studio-2021.3.1.17-windows.zip)
+
+## Configure Android Studio Environment Variables
+
+<aside>
```

Comment:
What's this? I don't see any formatting in the markdown.

https://github.com/HathorNetwork/hathor-wallet-mobile/blob/0becacf654599fbf8b88859a28054d912ce50021/WINDOWS.md